docker/Dockerfile.demo

```diff
@@ -1,16 +1,24 @@
-FROM python:3.9-slim-buster
+FROM python:3.10-slim-bookworm
 
 # Set the timezone
 RUN ln -fs /usr/share/zoneinfo/America/Los_Angeles /etc/localtime
 
 ENV DEBIAN_FRONTEND noninteractive
 # Default Python file.open file encoding to UTF-8 instead of ASCII, workaround for le-utils setup.py issue
 ENV LANG C.UTF-8
-RUN apt-get update && apt-get -y install python3-pip python3-dev gcc libpq-dev make git curl libjpeg-dev ffmpeg
-
-# install node
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && apt-get install -y nodejs
-
+RUN apt-get update && apt-get -y install python3-pip python3-dev gcc libpq-dev make git curl libjpeg-dev libssl-dev libffi-dev ffmpeg
+
+# Pin, Downloand and install node 16.x
+RUN apt-get update \
```

Comment:
Interested in the rationale for this update - does the previous nodesource install no longer work on newer images?

Reply:
The previous nodesource install script, installs node 18.x in the newer image. Hence this update

Reply:
Thanks, good to know! We'll hopefully upgrade to node 18 in the nearish future, so can probably revert to this method then.